aws/sdk/examples/iam/src/bin/iam-hello-world.rs

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0.
+ */
+
+use http::Uri;
+use iam::Endpoint;
+
+#[tokio::main]
+async fn main() -> Result<(), iam::Error> {
+    tracing_subscriber::fmt::init();
+    let ep = Endpoint::immutable(Uri::from_static("https://iam.amazonaws.com"));
```

Comment:
Should we add a comment here that the manual endpoint won't be necessary in the future with a link to the tracking issue?

Reply:
merging now so that I can add a test for this